docs/appendix/troubleshooting.md

```diff
@@ -3,227 +3,101 @@ title: "Troubleshooting"
+
+## Common Installation Errors
+
+### Verifying write failed. Contents of SD card is different from what was written to it.
+
+- **Description:** When following our Raspberry Pi [installation guide](/installation/prepare/rpi-setup/), the [Raspberry Pi imager](https://www.raspberrypi.com/software/) may fail to validate the written Raspberry Pi OS image when using some micro SD card readers or USB hubs.
+
+- **Solution:** Try a differnt USB port on your computer, or use a different micro SD card reader.
+   If you are connecting the micro SD reader through a USB hub, try connecting the reader directly to your computer instead.
+
+### The authenticity of host 'name-pi.local' can't be established.
+
+- **Description:** When following our [installation guides](/installation/), the very first `ssh` connection you make to your newly-imaged [board](/components/board/) (such as a Raspberry Pi) will likely encounter this message.
+   This is expected: `ssh` is advising you that it has not yet connected to this address, and prompts you for how to proceed.
+
+- **Solution:** The message will ask `Are you sure you want to continue connecting?`.
+   Type `yes` and then press the return key to continue with the connection.
+   You should recieve a successful confirmation message similar to: `Warning: Permanently added 'name-pi.local' to the list of known hosts.`
+   This is only required for the very first `ssh` connection you make to a newly-installed board.
+
+### ssh: connect to host name-pi.local port 22: Host is down
+
+- **Description:** Your computer is not able to connect to the [board](/components/board/) (such as a Raspberry Pi) via `ssh`.
+
+- **Solution:** Ensure that both your computer and the board itself are connected to the internet, and verify each of the following:
+   - If you are on a Windows computer, be sure that you do not have an outoging firewall preventing `ssh` connections over port `22`.
+   - Your `ssh` connection string should resemble the following: `ssh username@name-pi.local`. Be sure that you match hostname, username, and password exactly to what you initially configured when imaging your board.
+   - If you are still unable to connect, try power cycling the Pi and trying your connection again.
+   - If that fails, try re-imaging your board following the [installation guide](/installation/) appropriate for your board.
+   If using the Raspberry Pi [installation guide](/installation/prepare/rpi-setup/), be sure to carefully enter the configuration details under the **Advanced Options** (gear icon) button on the [Raspberry Pi imager](https://www.raspberrypi.com/software/) before you re-image your board.
+
+### ssh: connect to host name-pi port 22: Connection timed out
+
+- **Description:** Your computer is not able to connect to the [board](/components/board/) (such as a Raspberry Pi) via `ssh` before reaching a timeout.
```

Comment:
```suggestion
- **Description:** Your computer is not able to connect to the [board](/components/board/) through `ssh` before reaching a timeout.
```
[opt] We are trying not to use via or mention RPI over other boards, don’t think example is necessary here but up to you tbh